apps/prairielearn/elements/pl-sketch/pl-sketch.css

```diff
@@ -0,0 +1,74 @@
+.pl-sketch {
+  overflow: scroll;
+  width: 100%;
+  height: 100%;
+  padding: 5px 5px 5px 5px;
+}
+
+.pl-sketch-controls {
+  height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: #fafafa;
+  box-shadow: 0px 0px 2px #757575;
+  z-index: 2;
```

Comment:
Where possible, we prefer using Bootstrap classes to style elements. That means `d-flex`, `align-items-center`, and so on.